AmplifyPlugins/Storage/AWSS3StoragePlugin/Operation/AWSS3StorageUploadFileOperation.swift

```diff
@@ -98,10 +98,12 @@ public class AWSS3StorageUploadFileOperation: AmplifyInProcessReportingOperation
             return
         }
 
-        let prefixResolver = storageConfiguration.prefixResolver ??
-            StorageAccessLevelAwarePrefixResolver(authService: authService)
-        let prefixResolution = prefixResolver.resolvePrefix(for: request.options.accessLevel,
-                                                            targetIdentityId: request.options.targetIdentityId)
+        resolvePrefix(accessLevel: request.options.accessLevel, targetIdentityId: request.options.targetIdentityId) { prefixResolution in
```

Comment:
super nit: this seems quite long line, can we move the `targetIdentityId` argument on a new line under `accessLevel`?